README.md

```diff
@@ -29,7 +29,7 @@ Browse the code to see what it does, or build and run it yourself!
 
```

Comment:
In Running in Eclipse section:

1. Download and install [Eclipse with the WebSphere Developer Tools](https://developer.ibm.com/wasdev/downloads/liberty-profile-using-eclipse/).
 2. ~~Create a new Liberty Profile Server.~~
 3. Clone this repository.
 4. Import the sample into Eclipse using *File -> Import -> Maven -> Existing Maven Projects* option.
 5. ~~Edit the server.xml file located in the Servers -> DefaultServer folder in EnterpriseExporer. Copy the server.xml features from this project, found under the `<featureManager>` in /src/main/wlp/server.xml~~
 6. ~~Deploy the sample into Liberty server.~~ Right click on the project and select *Run As -> Run on Server* option. Find and select the Liberty profile server and press *Finish*.
 7. Go to: [http://localhost:9080/sample.javaee7.el30/](http://localhost:9080/sample.javaee7.el30/)